base/server/src/main/java/com/netscape/cms/password/PasswordChecker.java

```diff
@@ -76,16 +49,11 @@ public String getConfigReason(String mPassword) {
      */
     @Override
     public boolean isGoodPassword(String mPassword) {
-        if (mPassword == null || mPassword.length() == 0) {
-            return false;
-        } else if (mPassword.length() < MIN_LEN) {
-            return false;
-        }
-        return true;
+        return !(mPassword == null || mPassword.length() < MIN_LEN);
```

Comment:
The inverted expression might be easier to read:
```
return mPassword != null && mPassword.length() >= MIN_LEN;
```

Reply:
Agreed, new push includes this change,